apps/admin/controllers/moderation/update.rb

```diff
@@ -6,6 +6,7 @@ def call(params)
       if params[:action] == 'approve'
         TaskRepository.new.update(params[:id], { approved: true })
         ApproveTaskWorker.perform_async(params[:id])
+        ApprovedTaskTwitterWorker.perform_async(params[:id])
```

Comment:
can we use service instead twitter worker and put this service to `ApproveTaskWorker`?